script.js

```diff
@@ -10,219 +10,309 @@ menubar.addEventListener('click', () => {
 const openerButton2 = document.querySelector('.button2');
 openerButton2.addEventListener('click', () => {
   const main = document.createElement('div');
   main.className = 'main';
   const maincard = document.createElement('div');
   maincard.className = 'maincard';
   maincard.innerHTML = `
+
   <button class="close-button2">&times;</button>
   <div>
  
-  <img  class = "popup-image"src="images/pop-up/Snapshoot Portfolio.png" alt="">
+  <img  class = "popup-image"src="${array[0].mobileImg}" alt="">
+  <img  class = "popup-image2"src="${array[0].desktopImg}" alt="">
  </div>
-       <div class ="frame18"> <button class="first-button1" type="button" id="messagep">See source <img src="images/pop-up/button-icons/github.png" alt=""></button>
-        <button class="first-button2" type="button" id="messagep">See live <img src="images/pop-up/button-icons/Icon.png" alt=""></button>
-    <h3 class="popup-par">Keeping track of hundreds of <br> components</h3></div>
+ <div class="headline">
+    <h2 class="popup-par">${array[0].name}</h2>
+    <div class ="frame18">
+       <button class="first-button2" type="button" id="messagep">See live <img src="${array[0].liveImg}" alt=""></button>
+       <button class="first-button1" type="button" id="messagep">See source <img src="${array[0].githubImg}" alt=""></button>
+    </div>
+  </div>   
     <ul class="popup-list">
-      <li class="ruby">Codekit</li>
-      <li class="css">Github</li>
-      <li class="javascript">JavaScript</li>
-      <li class="html">Bootstrap</li>
-      <li class="html">Terminal</li>
-      <li class="html">Codepen</li>
+      <li class="ruby">${array[0].tech[0]}</li>
+      <li class="css">${array[0].tech[1]}</li>
+      <li class="javascript">${array[0].tech[2]}</li>
+      <li class="html">${array[0].tech[3]}</li>
+      <li class="html">${array[0].tech[4]}</li>
+      <li class="html">${array[0].tech[5]}</li>
     </ul>
-    <div class ="lorem"><h2>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea</h2></div>
-  `;
-  sectioncard2.appendChild(main);
-  sectioncard2.appendChild(maincard);
+    <div class ="lorem">${array[0].desc}</div>
+    <div class ="frame19"><button class="mobile-first-button1" type="button" id="messagep">See source <img class="btn-img" src="${array[0].githubImg}" alt=""></button>
+       <button class="mobile-first-button2" type="button" id="messagep">See live <img class="btn-img" src="${array[0].liveImg}" alt=""></button></div>
+    `;
+  // sectioncard2.appendChild(main);
+  // sectioncard2.appendChild(maincard);
```

Comment:
- [ ] [OPTIONAL]
There are commented code blocks found in your code on the repository. Kindly endeavor to remove **all** commented code from your code before pushing it to the repository so as to keep your code well organized.